hledger/test/journal/transaction-prices.test

```diff
@@ -210,7 +210,19 @@ hledger -f - balance --no-total -E
                  -1Y  b
 >>>= 0
 
-# 18. the above with -B
+# 18. The above, but hides accounts with zero amount but non-zero cost
```

Comment:
Or perhaps: 
Without -E, a should be hidden since its balance is zero - even though it has a non-zero cost.

Reply:
Sounds reasonable.